handsontable/src/pluginHooks.js

```diff
@@ -824,6 +824,8 @@ const REGISTERED_HOOKS = [
    *
    * `row` is a visual row index.
    *
+   * Note: Nullified array items or `false` return value can be used to disregard changes, as presented in the code snippet below
```

Comment:
```suggestion
   * __Note:__: Nullified array items or `false` return value can be used to disregard changes, as presented in the code snippet below.
```

Added the use of bold text, and a period at the end of the sentence, as seen in the other notes on this page.